### ISSUE TITLE
[Mac + Mono + Fiddler Error] Path to config file is not generated properly

### ISSUE THREAD

**Issue by sordidfellow** [cppisking/ffrk-inspector#81]
Aside: I realize that running the Fiddler extension on a Mac+Mono+Fiddler setup is not officially supported.  Just documenting this because it may be a super easy fix and maybe support will one day be given....  A man can dream.

When starting up Fiddler with the FFRK Inspector extension installed, it gives the following error message:
![screen shot 2015-06-09 at 10 35 25 pm](https://cloud.githubusercontent.com/assets/730944/8098947/f0f1866c-0fbb-11e5-944c-0a440e4154d8.png)

The generated path is `/Users/<username>/.local/share/Fiddler_evidencehere/1.0.0.0/ffrk_inspector_settings.config` but that path does not exist.  

On windows, this path is:
`C:\Users\<username>\AppData\Local\Telerik\Fiddler.exe_Url_wsd<random letters>\4.5.1.2\ffrk_inspector_settings.config`

Due to the broken path, there is a cascade of errors:
![screen shot 2015-06-09 at 10 35 35 pm](https://cloud.githubusercontent.com/assets/730944/8098978/743ad3fc-0fbc-11e5-91ef-19870c11fc3a.png)

![screen shot 2015-06-09 at 10 36 04 pm](https://cloud.githubusercontent.com/assets/730944/8099011/e2661986-0fbc-11e5-9af3-17d68a8ccaf4.png)

![screen shot 2015-06-09 at 10 36 26 pm](https://cloud.githubusercontent.com/assets/730944/8099014/e7ab172a-0fbc-11e5-9b18-8f078c2d867e.png)

**Comment by cppisking** [comment]
Can you try creating an empty file with that name and see what happens?

**Comment by sordidfellow** [comment]
Manually creating that path bypasses the first 2 errors and skips straight to this error:

![screenshot 2015-06-10 22 31 38](https://cloud.githubusercontent.com/assets/730944/8099336/250b7214-0fc1-11e5-901a-32d6dc4b3450.png)

![screenshot 2015-06-10 22 31 45](https://cloud.githubusercontent.com/assets/730944/8099338/2b35d42c-0fc1-11e5-82f7-467077156b42.png)



I copied a good file from a windows machine, and it had no effect...  But then I realized Fiddler hadn't crashed yet.  I entered the DB credentials and hit Test Connection and bam, it worked!

Looks like this setup isn't quite ready yet - Turning on "Accept Remote Connections" in the Fiddler preferences is crashing Fiddler 100% of the time.  But that looks like Mono's fault!

**Comment by sordidfellow** [comment]
TL;DR: Fiddler is still too flakey on my Mac for me to use.

Had to set preferences by editing the xml files that mono uses to simulate the registry.  Could not find a way to do a passthrough of traffic not headed for ffrk.denagames.com, so just starting the FFRK app was a huge pain - it usually hung indefinitely.  Toggling on and off captures in Fiddler to get past the FFRK login sequence frequently caused Fiddler to crash.

If anyone else is wanting to experiment with this, here's what I found:
* Any dialog popups besides the preference pane will usually crash Fiddler
* Since clicking on "Allow remote connections" and some of the other preferences causes a warning popup, you need to set those preferences outside of fiddler
* Those preferences are set in a virtual Windows Registry.  Mono maintains the registry in your ${HOME}/.mono folder.  The structure mimics an actual Windows Registry pretty well - just look for a values.xml file in the folder you navigate to and you can manually edit the file in any text editor.
* I tried setting Allow Remote Connection to Yes (basic requirement for our purposes), Ignore Cert Errors to Yes (b/c the mono version of fiddler always has a cert error popup), Be System Proxy to No (Fiddler barely functions, so why intercept all your Mac's traffic?  Turned it off), and maybe some others?
* The Cert Maker plugin that you download separately on windows is built-in to this version, so that part is actually easier, I think
* The config files generated by the FFRKInspector extension will go in ${HOME}/.local folder, which you have to manually create to prevent startup errors.

That's all I've got for now.  It just crashes too often to do much.

**Comment by cppisking** [comment]
Did you try the version of Fiddler that is supposed to be for Mono?
http://fiddler.wikidot.com/mono   No idea if this is officially maintained
or what, but it might be worth a try.

**Comment by sordidfellow** [comment]
That is the version I used.  Latest Mono + Fiddler Mono for Mac.  Last updated in 2013 :(

It looks like the Linux version is a lot newer - I can do without the attach/detach scripts and installer, so I might give that combo a shot tonight.
